parachains/runtimes/assets/statemine/src/weights/xcm/pallet_xcm_benchmarks_fungible.rs

```diff
@@ -51,14 +51,14 @@ impl<T: frame_system::Config> WeightInfo<T> {
 	// Storage: System Account (r:1 w:1)
 	// Proof: System Account (max_values: None, max_size: Some(128), added: 2603, mode: MaxEncodedLen)
 	pub(crate) fn withdraw_asset() -> Weight {
-		Weight::from_parts(22_035_000 as u64, 0)
+		Weight::from_ref_time(23_462_000 as u64)
```

Comment:
this is deprecated API, we need this PR https://github.com/paritytech/cumulus/pull/2344
the CI jobs will probably fail with this deprecated api